north/src/runtime/console.rs

```diff
@@ -234,4 +278,42 @@ async fn connection(stream: TcpStream, mut tx_main: EventTx) {
             break;
         }
     }
+
+    async fn create_installation_event<R: Unpin + Read>(
+        mut reader: R,
+        tmp_installation_dir: &std::path::Path,
```

Comment:
We should use `async_std::path` in the application. The `std::path` is only used in the settings because `async_std::path::*` doesn't impl `Serialize`/`Deserialize`. 
This will vanish with the `tokio` migration.

Reply:
same as above